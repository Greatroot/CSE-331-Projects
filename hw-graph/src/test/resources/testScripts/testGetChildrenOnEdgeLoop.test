# Tests to see if the graph stores loops between multiple nodes properly (ex: n1 -> n2, n2 -> n1).

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3

AddEdge g n1 n2 e1
AddEdge g n2 n1 e2

ListNodes g

ListChildren g n1
ListChildren g n2