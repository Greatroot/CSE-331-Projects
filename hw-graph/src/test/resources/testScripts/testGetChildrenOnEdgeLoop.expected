# Tests to see if the graph stores loops between multiple nodes properly (ex: n1 -> n2, n2 -> n1).

created graph g

added node n1 to g
added node n2 to g
added node n3 to g

added edge e1 from n1 to n2 in g
added edge e2 from n2 to n1 in g

g contains: n1 n2 n3

the children of n1 in g are: n2(e1)
the children of n2 in g are: n1(e2)